CONTRIBUTING.md

```diff
@@ -11,7 +11,8 @@ contributors to follow to reduce the time it takes to get changes merged in.
 2.  Ensure that your code blends well with ours:
     *   No trailing whitespace
     *   Match indentation (two spaces)
-    *   Match coding style (run `rake rubocop`)
+    *   Match coding style (run `rake rubocop:rubygems` or
+        `rake rubocop:bundler`)
```

Comment:
Can we instead make `rake rubocop` run both tasks, so we don't have to change this and we run lints for both repos by default. It's possible that PRs will change both repos, and I think it will get more common as we unify the libraries more and more.

Reply:
Nice, that'd be even better.